src/Microsoft.Data/DataFrame.cs

```diff
@@ -45,18 +45,9 @@ public DataFrame(IList<DataFrameColumn> columns)
 
         public int ColumnCount => _table.ColumnCount;
 
-        public IList<string> Columns
-        {
-            get
-            {
-                var ret = new List<string>(ColumnCount);
-                for (int i = 0; i < ColumnCount; i++)
-                {
-                    ret.Add(_table.Column(i).Name);
-                }
-                return ret;
-            }
-        }
+        public IReadOnlyList<DataFrameColumn> Columns => _table.Columns;
+
+        public IReadOnlyList<string> GetColumnNames() => _table.GetColumnNames();
```

Comment:
I thought we were going to leave this to the "Schema" property.

Reply:
Right, but without this method, there's be no way to examine the column names in the DataFrame. IDataView's `Schema` doesn't expose a way to get the column names(except by enumerating the `Schema's Columns`).

Reply:
> IDataView's Schema doesn't expose a way to get the column names(except by enumerating the Schema's Columns).

That `except` is exactly how you get the column names. You inspect the Schema. Having 2 ways of getting the column names is duplication. Why do we need this API when we already have a way of getting the column names (and even more information)?

Reply:
So, my reasoning was, in a notebook, if I wanted to see the column names, I'd have to write a foreach loop instead of 1 function call. I see now that executing `(IDataView)dataFrame.Schema` in a cell prints the column names and types. So, I'll pull this out.